### ISSUE TITLE
$(...).mCustomScrollbar is not a function with Aurelia Framework

### ISSUE THREAD

**Issue by JoshuaCE** [malihu/malihu-custom-scrollbar-plugin#428]
Hi, i have been trying to get the plugin work with Aurelia Framework.

installed plugin via JSPM package and imported into the page with "import 'malihu.......'
but getting error "$(...).mCustomScrollbar is not a function" when try to run the code.

**Comment by leemiser** [comment]
Having the same issue here.

**Comment by jordymeow** [comment]
Have you guys figured it out?

**Comment by JoshuaCE** [comment]
@jordymeow No.. Try Nicescroll. but it has its own issues. im using browser default scroll now.
